resources/assets/js/components/teams/TeamEditForm.vue

```diff
@@ -216,7 +216,8 @@ export default {
           this.hasError = false;
           this.feedback = 'Saved!';
           console.log('success');
-          window.location.href = '/admin/teams/' + response.data.team.slug;
+          var newHREF = '/admin/teams/' + response.data.team.slug;
+          if (!window.location.href.endsWith(newHREF)) window.location.href = newHREF;
```

Comment:
Why is the reload necessary?

Reply:
That form is also used for creating the team initially, so it'll redirect from `/admin/teams/create` to `/admin/teams/<slug>`. When editing an existing one it won't redirect now.

Reply:
Oh derp this is on the edit form, carry on.